libraries/abstractions/secure_sockets/include/iot_secure_sockets.h

```diff
@@ -257,6 +257,45 @@ Socket_t SOCKETS_Socket( int32_t lDomain,
                          int32_t lProtocol );
 /* @[declare_secure_sockets_socket] */
 
+/**
+ * @brief Bind a TCP socket.
+ *
+ * See the [FreeRTOS+TCP networking tutorial]
+ * (https://freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_TCP/TCP_Networking_Tutorial.html)
+ * for more information on TCP sockets.
+ *
+ * See the [Berkeley Sockets API]
+ * (https://en.wikipedia.org/wiki/Berkeley_sockets#Socket_API_functions)
+ * in wikipedia
+ *
+ * @sa SOCKETS_Bind()
+ * A pre-configured source port allows customers to bind to the specified local port instead of ephemeral port
+ * for security and packet filter reasons.
+ *
+ * Limitations:
+ *
+ *   i.  The caller of SOCKETS_Bind() API should make sure the socket address has the correct local IP address for the interface.
+ *   ii. The LWIP stack has ephemeral port range from 49152 to 65535, it is better avoid the source port in this range.
+ *
+ *       NOTE: If the SOCKETS_Bind() API binds to a source port in ephemeral port range and calls SOCKETS_Bind() API
+ *             but before SOCKETS_Connect() API is called then a conflict of source port arises that another TCP connection
+ *             may pick the the same chosen port via API tcp_new_port() ( by scanning its internal TCP connection list )
```

Comment:
This sentence in the note seems difficult to understand in the first row. Should it be something like this?
```suggestion
 *       NOTE: If the SOCKETS_Bind() API binds to a source port in ephemeral port range, and the caller calls SOCKETS_Bind() API
 *                  before SOCKETS_Connect() API ,then a conflict of source port arises as another TCP connection
 *             may pick the the same chosen port via tcp_new_port() API ( by scanning its internal TCP connection list )
```

Reply:
done